docs/services/navigation/_index.md

```diff
@@ -115,6 +119,54 @@ The following attributes are available for `Navigation` services:
+
+- Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
+  Scroll to the card with the name of your base:
+
+    {{< imgproc src="/services/navigation/click-add-frame-ui.png" alt="The button to add a frame selected with the cursor on the Viam App config builder." resize="500x" >}}
+
+- Click **Add Frame**.
+
+  - Keep the `parent` frame as `world`.
+  Select the **Geometry** drop-down menu.
+  - Configure a **Geometry** for the base that reflects its physical dimensions.
+  Reference [these instructions](/services/frame-system/#bounding-geometries) to configure your geometry and measure the physical dimensions of your base.
+
+    ![The frame card for the base in the Viam App config builder.](/services/navigation/full-frame-base-ui.png)
+
+- Scroll to the card with the name of your movement sensor.
+  Click **Add Frame** and select the **Parent** box.
+  - Type in the `name` of your base to specify this component as the `parent` of the sensor in the reference frame coordinate system, and click **Save Config** to save your configuration.
+  See [how to configure nested reference frames](/services/frame-system/nested-frame-config/) for an explanation of this configuration process.
+  ![The frame card for the base in the Viam App config builder.](/services/navigation/full-frame-movement-sensor-ui.png)
+
+  - Give the movement sensor a **Translation** that reflects where it is mounted on your base, measuring the coordinates with respect to the origin of the base.
+
+    In other words, designate the origin of the base as `(0,0,0)`, and measure the distance the sensor is *translated*, or, *displaced*, from that point, to obtain the coordinates of the **Translation**.
+
+    See [the Frame System Service](/services/frame-system/#configuration) for more information, and [Viam's Internals](/internals/orientation-vector/) for a detailed guide on conducting this measurement.
+
+Then, to calibrate your Frame System for the most accurate autonomous GPS navigation with the Navigation Service:
+
+- After configuring your robot, navigate to the **Control** page and select the card matching the name of your Navigation Service.
+- Monitor the readings displayed on the card, and verify that the compass or orientation readings from the movement sensor report `0` when the base is facing north.
+- If you cannot verify this:
+  - Navigate back to your robot's **Config** page. Scroll to the card with the name of your base. Adjust the **Orientation** of the frame to compensate for the mismatch.
+  - Navigate back to the Navigation card on your **Control** page, and confirm that the compass or orientation readings from the movement sensor now report `0` when the base is facing north, confirming that you've successfully calibrated your base to be oriented accurately within the Frame System.
+  - If you cannot verify this, repeat as necessary.
```

Comment:
```suggestion
  - If the movement sensor does not read `0` when the base is facing north, repeat as necessary.
```

Reply:
This makes sense procedurally but I don't think we need to say this x3!